src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -60,6 +64,7 @@
   private BeaconPerson owner;
 
   @Transient
+  @NotEmpty
```

Comment:
Ensures that the list is not empty when validating the JSON payload sent to the rest controller.

There are integration tests to check this case.